### ISSUE TITLE
Color contrast and dark mode accessibility

### ISSUE THREAD

**Issue by katemoser** [katemoser/one-thing#36]
# This is the issue description for the file /app/globals.css

 ### Accessibility Concerns:

 - The image tag in line 10 is missing an alt attribute. This is a problem because screen readers rely on the alt attribute to describe the content of the image to visually impaired users. To fix this issue, add an alt attribute to the image tag.
 yay!
